css/lp-archetypes.css

```
@@ -0,0 +1,3201 @@
+    max-width: 560px;
+    margin: 0 auto;
+  }
+
+  .lp-jp-bridge-arrow {
+    flex-direction: column;
+    justify-content: center;
+    margin: 16px auto;
+  }
+
+  .lp-jp-bridge-arrow span {
+    border-top: 13px solid currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)